Sources/Testing/Test.Case.Generator.swift

```diff
@@ -26,7 +26,7 @@ extension Test.Case {
     /// `Sequence` to allow the storage of computed sequences over collections
     /// (such as `CartesianProduct` or `Zip2Sequence`) that are safe to iterate
     /// multiple times.
-    private var _sequence: S
+    private var _sequence: @Sendable () async -> S
```

Comment:
I think this, and the modified parameters of the initializers below, might need to have the DocC updated to reflect that it's a function which returns the Sequence now. And maybe this could explain the reason why it's a closure

Reply:
Done.